resources/theme/index.ts

```diff
@@ -5,16 +5,22 @@
 import { extendTheme, ThemeOverride } from "@chakra-ui/react";
 import { mode, StyleFunctionProps } from '@chakra-ui/theme-tools';
 
-const Theme : ThemeOverride = {
+const Theme: ThemeOverride = {
     styles: {
         global: (props: StyleFunctionProps) => ({
             box: {
-                backgroundColor: mode("gray.50","backgroundDark")(props),
+                backgroundColor: mode("gray.50", "backgroundDark")(props),
             }
         })
     },
     colors: {
-        backgroundDark: "#1a202c"
+        blue: {
+            primary: "#4990FA"
+        },
+        dark: {
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
    dark: {
```